functions/openhab.bash

```diff
@@ -6,10 +6,9 @@
 ##    create_sys_dependencies()
 ##
 create_sys_dependencies() {
-  local targetDir
-
-  targetDir="/etc/systemd/system/openhab2.service.d"
+  local targetDir="/etc/systemd/system/openhab2.service.d"
 
+  echo -n "$(timestamp) [openHABian] Creating dependencies to jointly start services to depend on each other... "
```

Comment:
Should be "that depend" not "to depend".

Reply:
ok